packages/web-api-scan-runner/package.json

```diff
@@ -64,6 +64,7 @@
         "sha.js": "^2.4.11",
         "storage-documents": "1.0.0",
         "verror": "^1.10.0",
-        "yargs": "^13.2.1"
+        "yargs": "^13.2.1",
+        "markreay-accessibility-insights-report": "^0.0.3"
```

Comment:
Do we need to have this in both root and package specific level? Should it be only here? Need to be public version as well.

Reply:
Looks like it's fine with it only being in the package level. I've removed it from the root.